### ISSUE TITLE
Docker daemon 100% CPU utilization

### ISSUE THREAD

**Issue by netoneko** [orbs-network/boyarin#146]
Might be caused by this:

```
Apr  6 11:18:14 ip-172-31-1-207 dockerd[3381]: time="2020-04-06T11:18:14.117428878Z" level=error msg="logs call failed" error="failed getting container logs: No such container: a32884-chain-1004-stack.1.zp1ulq2v75a1z0pc1n3ujse3q" module=node/agent/taskmanager node.id=994b1qu79z1f0urrpncl9dmaa
Apr  6 11:18:14 ip-172-31-1-207 dockerd[3381]: time="2020-04-06T11:18:14.198789123Z" level=error msg="logs call failed" error="failed getting container logs: No such container: a32884-chain-2000-stack.1.sa5do3xvjst9aqy56zmfl3o1u" module=node/agent/taskmanager node.id=994b1qu79z1f0urrpncl9dmaa
Apr  6 11:18:15 ip-172-31-1-207 dockerd[3381]: time="2020-04-06T11:18:15.522344235Z" level=error msg="logs call failed" error="failed getting container logs: No such container: a32884-chain-1004-stack.1.zp1ulq2v75a1z0pc1n3ujse3q" module=node/agent/taskmanager node.id=994b1qu79z1f0urrpncl9dmaa
```

**Comment by netoneko** [comment]
Possibly related to nginx stdout logs.

**Comment by netoneko** [comment]
pprof profiles of the docker daemon

[profile.node1.gz](https://github.com/orbs-network/boyarin/files/4787224/profile.node1.gz)
[profile.node4.gz](https://github.com/orbs-network/boyarin/files/4787225/profile.node4.gz)

**Comment by netoneko** [comment]
Flame graphs before and after truncating nginx log:

<img width="1310" alt="Screen Shot 2020-06-16 at 17 55 42" src="https://user-images.githubusercontent.com/614125/84791013-baddbc00-affa-11ea-8d65-63ba2fcef76d.png">

<img width="1303" alt="Screen Shot 2020-06-16 at 17 55 49" src="https://user-images.githubusercontent.com/614125/84791003-b6b19e80-affa-11ea-9735-8498692412aa.png">

**Comment by netoneko** [comment]
Fixed by #152
